sentry-android-core/src/main/java/io/sentry/android/core/ManifestMetadataReader.java

```diff
@@ -86,6 +87,8 @@ final class ManifestMetadataReader {
 
   static final String PERFORM_FRAMES_TRACKING = "io.sentry.traces.frames-tracking";
 
+  static final String SENTRY_GRADLE_PLUGIN_INTEGRATIONS = "io.sentry.integrations";
```

Comment:
```suggestion
  static final String SENTRY_GRADLE_PLUGIN_INTEGRATIONS = "io.sentry.gradle-plugin-integrations";
```
to be more specific, would need to change in the SAGP PR as well